lib/components/chart_bar.dart

```diff
@@ -15,48 +15,54 @@ class ChartBar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return LayoutBuilder(
-      builder: (ctx, constraints) {
+      builder: (context, constraints) {
         return Column(
           children: [
             Container(
-                height: constraints.maxHeight *0.15,
-                child: FittedBox(child: Text('${value.toStringAsFixed(2)}'))),
+              height: constraints.maxHeight * 0.15, // Why 15%?
+              child: FittedBox(
+                child: Text(
+                  '${value.toStringAsFixed(2)}',
+                ),
+              ),
+            ),
```

Comment:
Quando for construir Widgets, tente sempre adicionar vírgular de formação, exemplo:

Ok:
```dart
AppBarTheme(
  textTheme: ThemeData.light().textTheme.copyWith(
    headline6: TextStyle(
      fontFamily: 'OpenSans',
      fontSize: 20,
      fontWeight: FontWeight.bold,
  ),
),
```

Ao invés de:
```dart
AppBarTheme(
  textTheme: ThemeData.light().textTheme.copyWith(
    headline6: TextStyle(
                      fontFamily: 'OpenSans',
                      fontSize: 20,
                      fontWeight: FontWeight.bold,
                    )))));
```